Mathlib/Topology/Algebra/Valued/ValuativeRel.lean

```diff
@@ -74,17 +74,32 @@ instance (priority := low) {R : Type*} [CommRing R] [ValuativeRel R] [UniformSpa
   «v» := valuation R
   is_topological_valuation := mem_nhds_zero_iff
 
+lemma v_eq_valuation {R : Type*} [CommRing R] [ValuativeRel R] [UniformSpace R]
+    [IsUniformAddGroup R] [IsValuativeTopology R] :
+    Valued.v = valuation R := rfl
```

Comment:
Do we really want to make `Valued` survive? Is this needed or are you adding it for general purposes?

Reply:
I am not sure if we want to keep it in the long run or not, but it seems to be useful in the transition phase. The instance is already there and I'm just adding a API lemma for it.

Reply:
I agree and I am not planning to add any new API. However this is the single API lemma that should always come with the instance (describing what the `v` field is). Without it that instance is absolutely useless unless one abuses defeqs.

Reply:
Which instance are you referring to exactly?

Reply:
The one just above it.
```
instance (priority := low) {R : Type*} [CommRing R] [ValuativeRel R] [UniformSpace R]
    [IsUniformAddGroup R] [IsValuativeTopology R] :
    Valued R (ValueGroupWithZero R) where
```